src/main/java/io/xpring/xrpl/model/XRPTransaction.java

```diff
@@ -0,0 +1,187 @@
+package io.xpring.xrpl.model;
+
+import com.google.protobuf.ByteString;
+import io.xpring.xrpl.TransactionType;
+import org.xrpl.rpc.v1.Payment;
+import org.xrpl.rpc.v1.Transaction;
+import org.immutables.value.Value;
+
+import javax.annotation.Nullable;
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * A transaction on the XRP Ledger.
+ *
+ * @see "https://xrpl.org/transaction-formats.html"
+ */
+// TODO(amiecorso): Modify this object to use X-Address format.
```

Comment:
Just double checking that you wanted to commit this TODO.

Reply:
Yes, there is a consistent TODO across all the SDKs and will address in future PR